bin/test.sh

```diff
@@ -64,7 +64,7 @@ setup_permissions() {
     if [ "$SECURITY" = "strict" ]; then
         # custom configuration to enable auth stuff:
         ${COMMONS_TOOLS_DIR}/setup_permissions.sh nobody "*" # spark's default service.role
-        ${COMMONS_TOOLS_DIR}/setup_permissions.sh nobody hdfs-role
+        ${COMMONS_TOOLS_DIR}/setup_permissions.sh root hdfs-role
```

Comment:
@mohitsoni, @susanxhuynh says you instructed her to setup these permissions for root in hdfs in strict mode.  Why?  Aren't root users forbidden in strict mode?